man/man1/ztest.1

```diff
@@ -147,6 +147,12 @@ ztest -f / -V -T 120
 
 .SH "ENVIRONMENT VARIABLES"
 .TP
+.B "ZFS_HOSTID=id"
+Use \fBid\fR instead of the SPL hostid to identify this host.  Intended for use
+with ztest, but this environment variable will affect any utility which uses
+libzpool, including \fBzpool(8)\fR.  Since the kernel is unaware of this setting
```

Comment:
OK, that seems like something that would be nice to clean up at some point.  But I doubt that `ZFS_HOSTID=` would actually affect zpool(8), since zpool(8) isn't using spl_get_hostid() (I think).